migrations/20220516100946_orders.js

```diff
@@ -13,6 +13,7 @@ exports.up = async (knex) => {
     def.string('purchaser', 48).notNullable()
     def
       .enu('status', ['Created', 'Submitted', 'Rejected', 'Amended', 'Accepted'], {
+        enumName: 'order_status',
```

Comment:
agreed, but migration wasn't working when I tried running. Basically it was failing to update my existing state due to order status annd recipe status being under the same name so this is more like a fix. I agree very bad practise and this would be out of the question for live/prod envs

Reply:
alright, that's fine do the modification. This isn't deployed yet so it's fine, just wanted to highlight